test/server/cards/03_TWI/units/PadawanStarFighter.spec.ts

```diff
@@ -0,0 +1,53 @@
+describe('Padawan Star Fighter', function () {
+    integration(function (contextRef) {
+        describe('Padawan Star Fighter\'s ability', function () {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        spaceArena: ['padawan-starfighter'],
+                        groundArena: ['duchesss-champion'],
+                        hand: ['obiwan-kenobi#following-fate', 'jedi-lightsaber']
+                    },
+                    player2: {
+                        groundArena: ['luke-skywalker#jedi-knight'],
+                        spaceArena: ['mining-guild-tie-fighter']
+                    }
+                });
+            });
+
+            it('should not have +1/+1 because no unit or upgrade with Force trait controlled by the player', function () {
```

Comment:
can you merge all tests in one ? it's not a good design but it's to preserve our free github action quota